client/volume_prune.go

```diff
@@ -21,6 +27,16 @@ type VolumePruneResult struct {
 
 // VolumesPrune requests the daemon to delete unused data
 func (cli *Client) VolumesPrune(ctx context.Context, opts VolumePruneOptions) (VolumePruneResult, error) {
+	if opts.All {
+		if _, ok := opts.Filters["all"]; ok {
+			return VolumePruneResult{}, errdefs.ErrInvalidArgument.WithMessage(`conflicting options: cannot specify both "all" and "all" filter`)
+		}
+		if opts.Filters == nil {
+			opts.Filters = Filters{}
+		}
+		opts.Filters.Add("all", "true")
```

Comment:
Ideally, the next step would be to make the API have a `all` field or query-string, instead of using a filter for this.